examples/chrome-extension/vendor/honeybadger.min.js

```diff
@@ -0,0 +1,1456 @@
+(function (global, factory) {
```

Comment:
I'm curious if there is a way to use honeybadger.min.js from our CDN? Either just load it from there directly in the script tag in the extension, OR add a little `wget` script to download it to `vendor/` (and maybe add a `.gitkeep` so that the directory already exists)?

Reply:
Basically, I want to reduce any long-term maintenance overhead related to example projects.

Reply:
I agree with you. Actually, I tried loading the file directly from our CDN, but ran into security issues with chrome extensions. Plus, it is not the recommended practice, as noted in the [How to comply with CSP] (https://developer.chrome.com/docs/apps/contentSecurityPolicy/#how) section of chrome development guides:

>All JavaScript and all resources should be local (everything gets packaged in your Chrome App).

I like the `wget` script. I can try that! Is there a CDN url that always points to the latest version?

Reply:
Done. I added a script to download (I used `curl`) the browser client and modified the `README.md` file. Let me know what you think.